### PR TITLE
updated doc

### PR DESCRIPTION
**Pull request by randymgeorge**

### CODE REVIEW

File: src/content/docs/start-here/kafka-event.md [hunk 1]
```diff
@@ -209,10 +209,11 @@ Example:
     "datacontenttype" : "application/json",
     "data" : {
          "metadata": {
-           "id" :  "A234-1234-1234",      
+           "id" :  "A234-1234-1234",  
+           "org_id" : "IdP's id for the organization",    
            "resource_type" : "k8s-cluster",
-           "last_reported" : "2018-04-05T17:31:00Z",
-           "workspace": "workspace name",
+           "updated_at" : "2018-04-05T17:31:00Z",
```
Comment (josejulio) [inline src/content/docs/start-here/kafka-event.md:215]: Should we also include the `created_at` in here?  It is something that should be available along with the `update_at` at this point ?

The same would apply for the `deleted_at` (include the create and update)

Comment (randymgeorge) [reply src/content/docs/start-here/kafka-event.md:215]: We could.  I am not aware of a requirement/use case that would require this in the event. Are you?  I can ask around. If not, I would suggest that we add it when needed.  As you know, adds are pretty easy to do. Modifications are much harder.  Deletes are the hardest.  Thus, if we put this in, we will be stuck with it for some time.

Comment (josejulio) [reply src/content/docs/start-here/kafka-event.md:215]: Not aware of any requirement- In that case it makes sense to just keep the specific timestamp and we can evaluate later if we want to include it.

File: src/content/docs/start-here/inventory-api.md [hunk 2]
```diff
@@ -31,13 +31,13 @@ Required data is marked with an asterisk (*).
...
-- ___first_reported_by___
-- ___last_reported_by___
+- ___created_at___
+- ___updated_at___
+- ___deleted_at___
    
-The only metadata that a Reporter can provide is: 
-- ___workspace*___
+The only metadata that a Reporter provides is: 
+- ___org_id*___
```
Comment (josejulio) [inline src/content/docs/start-here/inventory-api.md:39]: Are also including the `org_id` in the relationship metadata?

Comment (randymgeorge) [reply src/content/docs/start-here/inventory-api.md]: good catch.  FYI...I moved some things around to 1) mention what the reporter provides first followed by what the service adds (was inconsistent across objects) and 2) added org_id to relationship type along with some rewording.

**Pull request merged**